pkg/controller/node/controller_utils.go

```diff
@@ -70,6 +71,14 @@ func deletePods(kubeClient clientset.Interface, recorder record.EventRecorder, n
 			continue
 		}
 
+		// Set reason and message in the pod object.
+		if updatedPod, err := setPodTerminationReason(kubeClient, &pod, nodeName); err != nil {
```

Comment:
Yes, when we return false, we [retry right away with 0 delay](https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/node/nodecontroller.go#L391).